juliac.jl

```diff
@@ -50,45 +50,69 @@ Base.@ccallable function julia_main(args::Vector{String})::Cint
             arg_type = String
             metavar = "<file>"
             help = "start up with the given system image file"
+        "--precompiled"
+            arg_type = String
+            metavar = "{yes|no}"
+            range_tester = (x -> x ∈ ("yes", "no"))
+            help = "use precompiled code from system image if available"
+        "--compilecache"
+            arg_type = String
+            metavar = "{yes|no}"
+            range_tester = (x -> x ∈ ("yes", "no"))
+            help = "enable/disable incremental precompilation of modules"
+        "--home", "-H"
+            arg_type = String
+            metavar = "<dir>"
+            help = "set location of `julia` executable"
+        "--startup-file"
+            arg_type = String
+            metavar = "{yes|no}"
+            range_tester = (x -> x ∈ ("yes", "no"))
+            help = "load ~/.juliarc.jl"
```

Comment:
From the arg name, `"--startup-file"`, I would assume that this flag allows me to _set a custom_ startup file. Is that desirable? I would either consider changing the behavior to what I just described, or changing the name of the variable to something like `--use-startupfile`?